foursight_core/es_connection.py

```diff
@@ -165,17 +165,32 @@ def search(self, search, key='_source'):
         if err_msg:
             raise ElasticsearchException(message=err_msg)
         # In next line, PyCharm's linter wrongly worries that 'res' might not be reliably set above. -kmp 6-Jun-2022
-        return [obj[key] for obj in res['hits']['hits']] if len(res['hits']['hits']) > 0 else []  # noQA
+        total = res['hits']['total']
+        return [obj[key] for obj in res['hits']['hits']] if len(res['hits']['hits']) > 0 else [], total  # noQA
 
-    def get_result_history(self, prefix, start, limit):
+    def get_result_history(self, prefix, start, limit, sort = "timestamp.desc") -> [list, int]:
         """
         ES handle to implement the get_result_history functionality of RunResult
         """
+        sort_field = "uuid"
+        sort_order = "desc"
```

Comment:
Not an official python thing as far as I know, but I often use `'desc'` (single quotes) for tokens (what Lisp would call symbols) and `"desc"` for free text. They mean the same in Python, but sometimes it's easier to search for something if it's in a standard form. Typically one doesn't search for free text but one does search for tokens.

Reply:
@willronchetti might have an opinion here.

Reply:
Python itself doesn't seem to care, I prefer `'` for everything personally and try to stick to that in things I write at least

Reply:
I was wondering about this. I tend to try to use `"` everywhere, pretty much, hmm.